### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by silasbaur**
- fix video card
- remove auto-width
- fix song page videos

[Preview](https://deploy-preview-213--crdsmusic-int.netlify.app/)

### CODE REVIEW

File: _includes/_video-card.html [hunk 1]
```diff
@@ -1,10 +1,13 @@
-<a href="/videos/{{ video.slug }}" class="video-card">
+<a href="/videos/{{ video.slug }}" class="video-card {% if include.size == 'small' %}video-card-small{% endif %}">
   <div
     class="video-card-image"
     style="background-image:  url('{{ video.image.url | imgix: site.imgix }}?{{ site.imgix_params.placeholder_for_video }}');"
     data-optimize-img
   >
     <crds-icon name="play-thin" size="50" color="white"></crds-icon>
   </div>
+  {% if include.size == "small" %}
```
Comment (tcmacdonald) [inline _includes/_video-card.html:9]: What's this?

**Review by tcmacdonald — CHANGES_REQUESTED** [review]
Thanks @silasbaur. I don't think this is a good solution. Can we show "other videos" if there are no "associated videos" to show? What about a zero state message like "There are no videos associated with this song." or something.  Looking for options here, thoughts?

Glorious_Day_-_Live___Crossroads_Music

File: _includes/_video-card.html (hunk 1 above)
Comment (silasbaur) [reply _includes/_video-card.html:9]: There are two layouts for the video card. The ones on the homepage and video pages and then there are the ones on the song pages which are not as big and do not display a title

Comment (tcmacdonald) [reply _includes/_video-card.html:9]: Sorry, I could've been clearer. Why do you have a condition that does nothing?

```
{% if true %}
  <!-- do nothing? -->
{% else %}
  ...
{% endif %}
```

**Review by tcmacdonald — APPROVED** [review]

**Pull request merged**